fastlane_core/lib/fastlane_core/update_checker/changelog.rb

```diff
@@ -21,6 +21,11 @@ def show_changes(gem_name, current_version)
       end
 
       def releases(gem_name)
+        # We have to follow redirects, since some repos were moved
+        # away into a separate org
+        # Taken from https://github.com/excon/excon/issues/115#issuecomment-40647211
+        Excon.defaults[:middlewares] << Excon::Middleware::RedirectFollower
```

Comment:
Thanks for raising this, I agree that we shouldn't set this globally. I have the feeling introducing initializers would be over-engineering for our use case.

Instead how would you feel if I updated the code to set the middleware just for this request? This should easily be possible with `Excon`, and therefore we don't have this in our global state.